Library/Formula/open-mesh.rb

```diff
@@ -1,31 +1,46 @@
 require "formula"
 
 class OpenMesh < Formula
-  desc "Generic data structure to represent and manipulate polygonal meshes"
-  homepage "http://openmesh.org"
-  url "http://www.openmesh.org/media/Releases/3.3/OpenMesh-3.3.tar.gz"
-  sha1 "673677702a27a929c9124f8d4ea2b188a0500c50"
-
-  bottle do
-    cellar :any
-    sha256 "e4bc6932d6a585fa96b9cc8f09c3037d852dae79072a8feea47612ba48598212" => :yosemite
-    sha256 "7806df7d4f499a801350087017944a79365336aec3b266bb5bee2ef17ffa017a" => :mavericks
-    sha256 "0ca60ff704a7d442bdd432d9a63f58d2e453d787648f3831bcaa27ac2f5a1114" => :mountain_lion
-  end
-
-  head "http://openmesh.org/svnrepo/OpenMesh/trunk/", :using => :svn
-
-  depends_on "cmake" => :build
-  depends_on "qt"
-
-  def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args
-      system "make install"
+    desc "Generic data structure to represent and manipulate polygonal meshes"
+    homepage "http://openmesh.org"
+
+    url "http://www.openmesh.org/media/Releases/3.3/OpenMesh-3.3.tar.gz"
+    sha1 "673677702a27a929c9124f8d4ea2b188a0500c50"
+
+    bottle do
+        cellar :any
+        sha256 "e4bc6932d6a585fa96b9cc8f09c3037d852dae79072a8feea47612ba48598212" => :yosemite
+        sha256 "7806df7d4f499a801350087017944a79365336aec3b266bb5bee2ef17ffa017a" => :mavericks
+        sha256 "0ca60ff704a7d442bdd432d9a63f58d2e453d787648f3831bcaa27ac2f5a1114" => :mountain_lion
     end
-  end
 
-  test do
-    system "#{bin}/mconvert", "-help"
-  end
+    head "http://openmesh.org/svnrepo/OpenMesh/trunk/", :using => :svn
+
+    depends_on "cmake" => :build
+    depends_on "qt" => :optional
+
+    # For 3.3 version, when BUILD_APPS=OFF, the fixbundle.cmake will not be
+    # generated for a lacking cmake command `configure_file`. I patched the
+    # cmakelists.txt a little by adding the missing line.
+    patch do
+        url "https://gist.githubusercontent.com/autosquid/41741a5ff42ddb4d104c/raw/6c2b49b3b23bdbafbed053dc8dc54d3407cdfba1/CMakeLists.patch"
+        sha256 "cd7f8e58dce947f1680fc0cb10bacf208fc52b384025d1598eef31b9c6840528"
+    end
+
+    def install
+        mkdir "build" do
+            args = std_cmake_args
+            if build.with? "qt"
+                args << "-DBUILD_APPS=ON"
+            else
+                args << "-DBUILD_APPS=OFF"
+            end
+            system "cmake", "..", *args
+            system "make install"
+        end
+    end
+
+    test do
+        system "#{bin}/mconvert", "-help"
```

Comment:
Test is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/29242/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_open_mesh/